cupyx/fallback_mode/__init__.py

```diff
@@ -3,3 +3,8 @@
 
 # "NOQA" to suppress flake8 warning
 from cupyx.fallback_mode.fallback import numpy  # NOQA
+
+from cupyx.fallback_mode import utils
+
+
+utils._create_magic_methods()
```

Comment:
Why not call this function in `utils`?

Reply:
Okay, no worries. I'll call where the `ndarray` class is created. It was just personal preference.